sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -1419,6 +1429,35 @@ private static EventProcessorOptions CreateOptions(EventProcessorClientOptions c
             };
         }
 
+        /// <summary>
+        /// Creates, starts, and enriches processing diagnostics scope.
+        /// </summary>
+        /// <param name="eventData">The instance of <see cref="EventData"/> which is being processed.</param>
+        /// <returns>The instance of scope.</returns>
+        private DiagnosticScope StartProcessorScope(EventData eventData)
+        {
+            var diagnosticScope = ClientDiagnostics.CreateScope(DiagnosticProperty.EventProcessorProcessingActivityName, ActivityKind.Consumer, MessagingDiagnosticOperation.Process);
+            if (diagnosticScope.IsEnabled)
+            {
+                if (MessagingClientDiagnostics.TryExtractTraceContext(eventData.Properties, out var traceparent, out var tracestate))
+                {
+                    // set link in all cases
+                    diagnosticScope.AddLink(traceparent, tracestate);
+                    // parent is not required, but allowed and helps to correlate producer and consumers
```

Comment:
```suggestion

                    // Parent is not required, but allowed and helps to correlate producer and consumers.
                    
```

nit:  Adding whitespace for conformance with surrounding coding style.